openshift/roles/pulp/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: 'Install packages'
+  yum:
+    name: "{{ item }}"
+    state: "latest"
+    update_cache: yes
+  with_items:
+    - pulp-server
+    - python-qpid
+    - python-qpid-qmf
+    - qpid-tools
+    - python-gofer-qpid
+    - pulp-rpm-plugins
+    - pulp-puppet-plugins
+    - pulp-docker-plugins
+    - pulp-katello
+
+- name: 'Add logs.py patch'
+  get_url:
+    url: https://raw.githubusercontent.com/ehelms/pulp/log-to-file/server/pulp/server/logs.py
```

Comment:
A community user actually added this feature in Pulp as of 2.13 and since this pulls from nightly I could not work on updating this.